src/server/datasource-config.ts

```diff
@@ -15,7 +13,6 @@ export const AppDataSource = new DataSource({
   password: process.env.DB_PASSWORD,
   database: process.env.DB_NAME,
   entities: [Chat, Message],
-  synchronize: true,

```

Comment:
This setting is dangerous once the app is in prod. Better to rely on migrations. See [here ](https://orkhan.gitbook.io/typeorm/docs/migrations#how-migrations-work)